src/job.rs

```diff
@@ -601,109 +611,108 @@ pub fn job_action_exists(context: &Context, action: Action) -> bool {
 
 /* special case for DC_JOB_SEND_MSG_TO_SMTP */
 #[allow(non_snake_case)]
-pub fn job_send_msg(context: &Context, msg_id: u32) -> libc::c_int {
-    let mut success = 0;
+pub fn job_send_msg(context: &Context, msg_id: u32) -> Result<(), Error> {
+    let mut mimefactory = MimeFactory::load_msg(context, msg_id)?;
```

Comment:
due to this "?" which bubbles up Errors, indentation in the function below changes. i'll mark the actual non-indent related changes